vaadinfrontend/frontend/themes/datamanager/components/dialog.css

```diff
@@ -0,0 +1,57 @@
+.dialog {
+}
+
+.dialog::part(overlay) {
+  width: 66vw;
+}
+
+.dialog .confirm-button {
+  background-color: var(--lumo-primary-color);
+  color: var(--lumo-primary-contrast-color);
+  font-weight: 600;
+  min-width: calc(var(--lumo-button-size) * 2.5);
+}
+
+.dialog .confirm-button:hover::before {
+  opacity: 0.05;
+}
```

Comment:
Major NIT: I'd recommend a small comment here to show that this is the vaadin button primary style.